eth_verifier/src/Verifier.sol

```diff
@@ -71,8 +71,8 @@ contract KimchiVerifier {
         // URS deserialization is WIP, we'll generate a random one for now:
         Scalar.FE x = Scalar.from(42);
         uint256 max_domain_size = 16384;
```

Comment:
If this var is not used, please remove it.

Reply:
You're right! though I'm actually reusing this variable in the PR that follows this one, I would leave it personally